libs/hdf-converters/src/converters-from-hdf/xccdf/reverse-xccdf-mapper.ts

```diff
@@ -0,0 +1,170 @@
+import {ExecJSON} from 'inspecjs';
+import moment from 'moment';
+import Mustache from 'mustache';
+import {version as HeimdallToolsVersion} from '../../../package.json';
+import {
+  MappedXCCDFtoHDF,
+  TestResultEnum
+} from '../../../types/reverseMappedXCCDF';
+
+function getXCCDFResult(control: ExecJSON.Control): TestResultEnum {
+  if (control.results.some((result) => result.backtrace)) {
+    return 'error';
+  }
+  if (control.results.some((result) => !result.status)) {
+    return 'unknown';
+  }
+
+  if (control.results.every((result) => result.status === 'passed')) {
+    return 'pass';
+  }
+
+  if (control.results.every((result) => result.status === 'skipped')) {
+    return 'notchecked';
+  }
+
+  if (control.results.some((result) => result.status === 'failed')) {
+    return 'fail';
+  }
+
+  return 'unknown';
+}
+
+function getXCCDFResultMessageSeverity(segments: ExecJSON.ControlResult[]) {
+  if (segments.some((result) => result.backtrace)) {
+    return 'medium';
+  }
+  return 'info';
```

Comment:
either this is wrong or the types is wrong between info or informational

Reply:
Please explain what you mean by this, does my previous comment address this?